packages/core/src/App/Containers/Layout/header/dtrader-v2-contract-detail-header.tsx

```diff
@@ -29,9 +28,9 @@ const DTraderContractDetailsHeader = observer(() => {
                     data-testid='arrow'
                     onClick={handleBack}
                 />
-                <Text size='md' bold>
+                <div className='contract-details-header-v2__content'>
```

Comment:
The issue is that styles was overwritten because of the loading order of css. When switching to Contract Details, we lazy-load component with quill styles (in Network a new chunk is added). As they arrive later, they overwrite some original css properties.
I tried to remove lazy-loading of Contract Details header and it actually fixed the bug with styles overwritting, buuut that cause another issue: during first load and reloding, black blink appears.
I also tried to add webpackExclude comment with reg exp for quill, but it didn't worked.